src/main/java/io/stargate/sgv2/jsonapi/util/SortClauseUtil.java

```diff
@@ -8,10 +8,20 @@
 public class SortClauseUtil {
   public static List<FindOperation.OrderBy> resolveOrderBy(SortClause sortClause) {
     if (sortClause == null || sortClause.sortExpressions().isEmpty()) return null;
+    if (sortClause.sortExpressions().stream().findFirst().get().path().equals("$vector"))
```

Comment:
This might be more readable by adding convenience method in `SortClause` (something like `hasVectorSort()` or whatever), called from here.